gerber/layers.py

```diff
@@ -28,62 +28,62 @@
 
 hints = [
     Hint(layer='top',
-         ext=['gtl', 'cmp', 'top', ],
-         name=['art01', 'top', 'GTL', 'layer1', 'soldcom', 'comp', 'F.Cu', ],
+         ext=['gtl', 'cmp', 'top', 'gbr', ],

```

Comment:
'gbr' should not be added as an extension hint to any of these layer types because it is not unique to any specific layer type.

Reply:
Yes, that's true that 'gbr' is not the unique extension, in practice, I should not add it as hint extension.